review_checklist.md

```diff
@@ -0,0 +1,35 @@
+# Review Checklist
+
+When reviewing, keep in mind that we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our contributions and interactions within this repository.
+
+**General tips**
+- Try create a culture of collaboration, when reviewing
```

Comment:
I'm not sure about this change myself though, but I think "try create" is not correct.
```suggestion
- Try to create a culture of collaboration when reviewing
```

Reply:
Good spot, thanks! committed